opentelemetry-jaeger/src/collector.rs

```diff
@@ -1,88 +1,247 @@
 //! # HTTP Jaeger Collector Client
-use crate::thrift::jaeger;
-use http::{Request, Uri};
-use isahc::{
-    auth::{Authentication, Credentials},
-    config::Configurable,
-    HttpClient,
-};
-use std::io::{self, Cursor};
-use std::sync::atomic::{AtomicUsize, Ordering};
-use thrift::protocol::TBinaryOutputProtocol;
+use http::Uri;
+use std::sync::atomic::AtomicUsize;
 
 /// `CollectorAsyncClientHttp` implements an async version of the
 /// `TCollectorSyncClient` interface over HTTP
 #[derive(Debug)]
 pub(crate) struct CollectorAsyncClientHttp {
     endpoint: Uri,
-    client: HttpClient,
+    #[cfg(feature = "collector_client")]
+    client: isahc::HttpClient,
+    #[cfg(feature = "wasm_collector_client")]
```

Comment:
These features have to be exclusive or else `--all-features` won't compile

Reply:
Thanks. I now enable the wasm feature only if the bare feature is not set.